src/storage/slotted_page.c

```diff
@@ -233,6 +234,7 @@ static int spage_put_helper (THREAD_ENTRY * thread_p, PAGE_PTR pgptr, PGSLOTID s
 			     const RECDES * recdes, bool is_append);
 static void spage_add_contiguous_free_space (PAGE_PTR pgptr, int space);
 static void spage_reduce_contiguous_free_space (PAGE_PTR pgptr, int space);
+static bool spage_skip_save_space_altogether ();
```

Comment:
Nitpicking: if you also agree a name like spage_is_save_space_enabled/disabled, please rename/update the code.

Reply:
Not nitpicking. It was a bad name.
Fixed.